internal/auth/authenticator_oidc.go

```diff
@@ -252,13 +308,17 @@ func extractDomainFromHost(host string) string {
 
 // Authenticate treat an authentication request coming from HAProxy
 func (oa *OIDCAuthenticator) Authenticate(msg *spoe.Message) (bool, []spoe.Action, error) {
-	ssl, host, pathq, cookieValue, err := extractOAuth2Args(msg)
+	ssl, host, pathq, clientid, clientsecret, redirecturl, cookieValue, err := extractOAuth2Args(msg, oa.options.ReadClientInfoFromMessages)
```

Comment:
Is this all the info w"re gonna get from this or we might add more client related info in the near future ?
If so, wrap this into a struct

Reply:
Done